### ISSUE TITLE
Rename the operator in OLM

### ISSUE THREAD

**Issue by vpavlin** [kubeflow/kfctl#229]
I have 2 issues with the recently merged PR for operator CSV

1. The name of the operator should be `kubeflow-operator` not `kfctl` - the whole thing should be probably renamed to keep things consistent with the name of the deployment 

   Right now the operator shows up as `kfctl` in OperatorHub - https://operatorhub.io/operator/kfctl, it would be best to make things consistent and rename it to kubeflow-operator

2. Would it make more sense to use `Kubeflow` as a provider of community operator (https://github.com/kubeflow/kfctl/blob/master/deploy/olm-catalog/kfctl/0.1.0/kfctl.v0.1.0.clusterserviceversion.yaml#L92)? Since it is going to be maintained under the Kubeflow project and community, I think that would make perfect sense

@animeshsingh @Tomcli @jlewi @pdmack Thoughs?

**Comment by pdmack** [comment]
Agreed @vpavlin 
Also, the metadata should use the proper case of "Kubeflow"

`Check [KubeFlow Operator documentation]`

**Comment by adrian555** [comment]
I agree to the changes suggested above and I will work on PRs to both repos.

**Comment by vpavlin** [comment]
1. addressed by https://github.com/kubeflow/kfctl/pull/270, thanks @adrian555!

**Comment by vpavlin** [comment]
@animeshsingh Any thoughts on 2.? Do you want to keep IBM in there?

**Comment by jlewi** [comment]
I'll defer to whoever is developing the operator about what the correct values should be.
